internal/runners/packages/list.go

```diff
@@ -154,12 +189,23 @@ type packageRow struct {
 	Version string `json:"version" locale:"package_version,Version"`
```

Comment:
Would be good if we can include the resolved version in the structured output as well.

Reply:
That's a good point. We can display the "Auto -> 1.1.1" for printed output and keep separate "Auto" and "1.1.1" fields for structured output.